### PR TITLE
fix #80595 3dnews.ru

### LINKED ISSUE #80595: 3dnews.ru
### Issue URL (Ads)
 [https://3dnews.ru/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2F3dnews.ru%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.1.1
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by stanislav-atr**
#80595

### CODE REVIEW

**Comment by Sergey-Lyapin** [comment]
not reproduced

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -95,6 +95,7 @@ rmnt.ru##div[id^="ad-in-content-"]
 ideaprog.download##.new_yapr
 ||kod.ru/assets/images/promo/
 starhit.ru##.hb-placeholder
+||aio.media/*/requestAd?appid=
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt:98]: It is ad networs. Better block `static.aio.media` in ad servers.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply RussianFilter/sections/specific.txt:98]: @stanislav-atr

Comment (stanislav-atr) [reply RussianFilter/sections/specific.txt]: @Alex-302, fixed.

**Review by zloyden — APPROVED** [review]

**Pull request merged**